components/clients/nova/Button.module.scss

```diff
@@ -0,0 +1,135 @@
+.button{
+  cursor: pointer;
+  border:none;
+  width: 148px;
```

Comment:
The width here doesn't seem correct, makes the button less usable, lets take this out and go with `width: 100%` and let the parent element decide the width, or pass it as a prop `style={{ width: '148px` }}